pkg/setting/setting.go

```diff
@@ -172,7 +173,7 @@ func init() {
 	logger = log.New("settings")
 }
 
-func parseAppUrlAndSubUrl(section *ini.Section) (string, string) {
+func parseAppUrlAndSubUrl(section *ini.Section) (string, string, string) {
```

Comment:
this seems a bit strange, why mingle the static url parsing with the static url?